include/miopen/miopen.h

```diff
@@ -573,6 +581,8 @@ typedef enum
     MIOPEN_CONVOLUTION_ATTRIB_DETERMINISTIC =
         1, /*!< Restrict MIOpen convolutions to kernels which produce numerically deterministic
               results. 0 - disabled (default), 1 - enabled >*/
+    MIOPEN_CONVOLUTION_ATTRIB_FP8_ROUNDING_MODE = 2,
```

Comment:
```suggestion
    MIOPEN_CONVOLUTION_ATTRIB_F8_ROUNDING_MODE = 2,
```
because this is applicable to both FP8 and BF8.